master/static/srv/get_trial.sql

```diff
@@ -31,7 +31,7 @@ FROM
                       c.labels
                FROM checkpoints c
                WHERE c.trial_id = t.id
-                 AND c.step_id = s.id ) r3) ASCHECKPOINT,
+                 AND c.step_id = s.id ) r3) AS CHECKPOINT,
```

Comment:
Was this typo causing a failure before?

Reply:
Funny enough, it didn't. It just set the name of the field to ASCHECKPOINT instead of CHECKPOINT.

Reply:
Nice